src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj

```diff
@@ -16,6 +16,7 @@
     <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
     <NoSemVer20>true</NoSemVer20>
     <IsPackable>true</IsPackable>
+    <IsShipping>false</IsShipping>
```

Comment:
Up to you but might use a condition like ` '$(PreReleaseVersionLabel)' == 'preview' ` or ` '$(StabilizePackageVersion)' != 'true' ` to make the switch automatic.l

Reply:
That sounds nicer :)